Failing circle test - "firefox on any platform - Firefox Desktop Sync v1 reset_password - reset password, verify different browser - from original tab's P.O.V."

Against current master, every time:

```
AssertionError: expected true to be false
  at Assertion.<anonymous>  <node_modules/intern/browser_modules/chai/chai.js:589:10>
  at Assertion.addProperty  <node_modules/intern/browser_modules/chai/chai.js:4240:29>
  at Function.assert.isFalse  <node_modules/intern/browser_modules/chai/chai.js:2521:31>
  at Command.<anonymous>  <tests/functional/lib/fx-desktop.js:80:20>
  at runCallback  <node_modules/intern/node_modules/leadfoot/Command.js:526:31>
  at Command.<anonymous>  <node_modules/intern/node_modules/leadfoot/Command.js:543:11>
  at <node_modules/intern/node_modules/dojo/Promise.ts:393:15>
  at run  <node_modules/intern/node_modules/dojo/Promise.ts:237:7>
  at <node_modules/intern/node_modules/dojo/nextTick.ts:44:3>
  at nextTickCallbackWith0Args  <node.js:420:9>
  at Command.then  <node_modules/intern/node_modules/leadfoot/Command.js:542:10>
  at Command.testIsBrowserNotifiedOfLogin  <tests/functional/lib/fx-desktop.js:71:10>
  at Command.<anonymous>  <tests/functional/lib/helpers.js:1310:25>
  at runCallback  <node_modules/intern/node_modules/leadfoot/Command.js:526:31>
  at Command.<anonymous>  <node_modules/intern/node_modules/leadfoot/Command.js:543:11>
  at <node_modules/intern/node_modules/dojo/Promise.ts:393:15>
  at run  <node_modules/intern/node_modules/dojo/Promise.ts:237:7>
  at <node_modules/intern/node_modules/dojo/nextTick.ts:44:3>
  at nextTickCallbackWith0Args  <node.js:420:9>
  at process._tickCallback  <node.js:349:13>
  at Command.then  <node_modules/intern/node_modules/leadfoot/Command.js:542:10>
  at Test.registerSuite.reset password, verify different browser - from original tab's P.O.V. [as test]  <tests/functional/sync_reset_password.js:134:10>
  at <node_modules/intern/lib/Test.js:181:24>
  at <node_modules/intern/browser_modules/dojo/Promise.ts:393:15>
  at runCallbacks  <node_modules/intern/browser_modules/dojo/Promise.ts:11:11>
  at <node_modules/intern/browser_modules/dojo/Promise.ts:317:4>
  at run  <node_modules/intern/browser_modules/dojo/Promise.ts:237:7>
  at <node_modules/intern/browser_modules/dojo/nextTick.ts:44:3>
  at nextTickCallbackWith0Args  <node.js:420:9>
  at process._tickCallback  <node.js:349:13>
```

--- a/tests/functional/sync_reset_password.js
+++ b/tests/functional/sync_reset_password.js
@@ -131,12 +131,9 @@ define([
         .then(type('#password', PASSWORD))
         .then(click('button[type=submit]'))
 
-        .then(testIsBrowserNotifiedOfLogin(this, email, { checkVerified: false }))
-
-        // user verified the reset password in another browser, they must
-        // re-verify they want to sign in on this device to avoid
-        // opening up an attack vector.
-        .then(testElementExists('#fxa-confirm-signin-header'));
+        // User verified from the same IP address so does not
+        // have to go through signin confirmation.
+        .then(testIsBrowserNotifiedOfLogin(this, email, { checkVerified: true }));
     },
 
     'reset password, verify different browser - from new browser\'s P.O.V.': function () {

